dace/codegen/instrumentation/likwid.py

```diff
@@ -82,38 +82,28 @@ def on_sdfg_begin(self, sdfg, local_stream, global_stream, codegen):
 std::string execpid = std::to_string(getpid());
 setenv("LIKWID_PERF_PID", execpid.c_str(), 1);
 
-int num_threads = 1;
-int num_procs = 1;
-#pragma omp parallel
-{{
-    #pragma omp single
-    {{
-        num_threads = omp_get_num_threads();
-        num_procs = omp_get_num_procs();
-    }}
+int num_threads = 0;
+if (getenv("OMP_NUM_THREADS") != NULL) {{
```

Comment:
is LIKWID_NUM_THREADS a real envvar? just asking from the previous review

Reply:
I removed LIKWID_NUM_THREADS again in the second commit, didn't squash it because I wanted to have the history. Now, it's only OMP_NUM_THREADS or max threads if not defined